internal/host/transition.go

```diff
@@ -60,7 +60,9 @@ func (th *transitionHandler) PostRegisterHost(sw stateswitch.StateSwitch, args s
 	if _, err := common.GetHostFromDB(params.db, hostParam.InfraEnvID.String(), hostParam.ID.String()); err == nil {
 		// The reason for the double register is unknown (HW might have changed) -
 		// so we reset the hw info and progress, and start the discovery process again.
+		// also log info is not current and should be resetted
 		extra := append(resetFields[:], "discovery_agent_version", params.discoveryAgentVersion, "ntp_sources", "")
+		extra = append(extra, resetLogsField...)
```

Comment:
i think we should move to use https://github.com/openshift/assisted-service/blob/master/internal/host/hostutil/update_host.go#L18 and remove "resetLogsField"

Reply:
The one in the host_utils is not used and should be removed